stores/s3store/sent_test.go

```diff
@@ -16,6 +16,7 @@ package s3store
 
```

Comment:
Can you create a `store_test.go` and move the tests that relate to `S3Store` in there. It will make it easier to find the tests and maintain